### DISCUSSION TITLE
Clarification on "Only N-1 transmission contingencies are supported"

### DISCUSSION THREAD

**Discussion by mzy2240** [ANL-CEEESA/UnitCommitment.jl#34] [Help & FAQ]
From the document, it seems we could define contingency with multiple lines open. In this case, will this contingency be checked to ensure the line limit won't be violated? Also what's the reason why only N-1 transmission contingencies are supported?

**Comment by iSoron** [comment]
Hi Zeyu, our **data format** allows arbitrary line and generator contingencies (for example, you could have a contingency simultaneously affecting two lines and a generator), but our **currently implemented algorithms** can only handle single-line contingencies. The reason is, we currently use Line Outage Distribution Factors (LODFs) to quickly evaluate post-contingency line flows, but these can only handle the removal of a single line. Some extensions to multiple lines have been proposed in the literature, but we haven't implemented them yet. This would be a great contribution to the package!
